bridge-stp.in

```diff
@@ -52,7 +58,7 @@ case $2 in
         if [ -e "$conf_file" ]; then
             . "$conf_file" || exit
         fi
-        checkpid $pid_file || exit 1
+        checkpid || exit 1
```

Comment:
I removed the argument because the original `checkpid` code was ignoring it when it actually read the PID file.  I will put it back and fix the code so it uses the argument.